### ISSUE TITLE
Colour contrast not accessible for code examples in the component guide

### ISSUE THREAD

**Issue by MartinJJones** [alphagov/govuk_publishing_components#3018]
The code example shown in the screenshot below does not have sufficient colour contrast.

<img width="800" alt="Screenshot 2022-10-13 at 11 40 40" src="https://user-images.githubusercontent.com/28779939/196132819-dd8b58a9-3acc-484d-a8fd-9a17777cd305.png">

We can fix this by using the same color and background-color used in the Design System for code examples:

```CSS
    color: #d13118;
    background-color: #f3f2f1;
```
https://design-system.service.gov.uk/components/back-link/
